docker/Dockerfile.worker

```diff
@@ -108,6 +108,7 @@ RUN MOUNT_S3_URL=$(if [ "$TARGETARCH" = "arm64" ]; then echo $MOUNT_S3_URL_ARM64
     mv mount-s3/bin/mount-s3 /usr/local/bin/ms3 && chmod +x /usr/local/bin/ms3 && \
     rm -rf mount-s3 $TARBALL
 
+RUN curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
I checked the script and my assumption is that this url points to the latest version of uv. Can we find a way to pin the version?

Reply:
I think we can probably just pin to a version in their [releases](https://github.com/astral-sh/uv/releases)

Reply:
Oh they actually have this ready to go 

curl --proto '=https' --tlsv1.2 -LsSf https://github.com/astral-sh/uv/releases/download/0.4.26/uv-installer.sh | sh

Reply:
Yeah that would work.